.github/workflows/code-style.yml

```diff
@@ -21,22 +21,33 @@ jobs:
 
       - name: Check Out Code
         uses: actions/checkout@v4
-        with:
-          fetch-depth: 1
 
-      - name: Get Composer Cache Directory
-        id: composer-cache
-        run: echo "::set-output name=dir::$(composer config cache-files-dir)"
```

Comment:
`ramsey/composer-install@v3` will handle the cache automatically. 

See https://github.com/ramsey/composer-install?tab=readme-ov-file#about